packages/node/README.md

```diff
@@ -181,7 +181,7 @@ if (module.hot) {
 }
 
 createServer((req, res) => {
-  res.end(app('example'));  
```

Comment:
Could you make this whitespace change to the `docs/` version too? :-)

Reply:
Done.